scripts/HealthCheck.java

```diff
@@ -0,0 +1,25 @@
+package uk.gov.digital.ho.hocs;
+
+import java.io.IOException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+
+public class HealthCheck {
+
+    public static void main(String[] args) throws IOException, InterruptedException {
+        var client = HttpClient.newHttpClient();
+        var request = HttpRequest.newBuilder()
+                .uri(URI.create("http://localhost:8080/actuator/health"))
```

Comment:
Will this always be 8080?

Reply:
8080 is the spring boot default so unless someone goes and changes it on purpose yes. This doesn't tie us to a port any more than having it defined as 8080 in the curl based health check